src/main/scala/is/hail/keytable/KeyTable.scala

```diff
@@ -322,6 +322,12 @@ case class KeyTable(hc: HailContext, rdd: RDD[Row],
     filter(p)
   }
 
+  def head(n: Long = 10): KeyTable = {
+    if (n < 0)
```

Comment:
the constraint doesn't match the error message. Should be either <= 0 or the message should say non-negative